### DISCUSSION TITLE
Next release 0.13 is coming end of this week

### DISCUSSION THREAD

**Discussion by rurban** [LibreDWG/libredwg#930] [General]
Only minor cleanup needed.
frank's copyright assignment is not here yet, so his PR"s are out for now

**Comment by Mannshoch** [comment]
could you may compile binarys for Linux?
I'm not that skilled in compiling and it seems I'm not able to get it done without errors.

**Reply by rurban** [reply]
which linux exactly?

**Reply by Mannshoch** [reply]
Ubuntu 23.10 running on a VMware VM
I'm very limited what I'm allowed to do on Windows.

**Reply by rurban** [reply]
Opensuse has a libredwg package. https://software.opensuse.org//download.html?project=openSUSE%3AFactory&package=libredwg
I"ll see what binary packages they have for this in their obs.

**Comment by rurban** [comment]
And now I've deleted all outdated nightlies and pre-releases, and its tags.
